### PR TITLE
[#8157] public logger attributes documentation

### PR DESCRIPTION
**Pull request by bennr01**
https://twistedmatrix.com/trac/ticket/8157

This pull request improves the documentation of `twisted.logger.Logger` by adding documenting `namespace`, `observer` and `source` as `@ivar`s.

This is my first pull request to twisted. I hope i am doing everything correctly ;)

~~**Edit:** Sorry, but i am unable to edit the ticket to provide a link to this pull request or changing its status. I am getting the error `Missing or invalid form token. Do you have cookies enabled?` (by the way, i have cookies enabled).~~

### CODE REVIEW

File: src/twisted/logger/_logger.py [hunk 1]
```diff
@@ -19,6 +19,15 @@ class Logger(object):
     A L{Logger} emits log messages to an observer.  You should instantiate it
     as a class or module attribute, as documented in L{this module's
     documentation <twisted.logger>}.
+
+    @type namespace: L{str}
+    @ivar namespace: the namespace for this logger
```
Comment (adiroiban) [inline src/twisted/logger/_logger.py]: we already have a good dosctring from this here https://github.com/twisted/twisted/pull/780/files#diff-88ae4a9d01a009bd424a1764a2ba5319R46

why add a different documentation?

**Comment by bennr01** [comment]
@adiroiban Thanks for the review. The parameters `namespace`, `source` and `observer` are indeed already documented, but not as `@ivar`s. Adding `@ivar foo: bar` in the docstring of the class makes reading the documentation a bit easier (the attributes are now mentioned in the API-documentation). By the way, the ticket was created by @glyph, so he may be able to explain it better than me.

**Comment by adiroiban** [comment]
sorry for the bad communication. Please see my comment from  https://twistedmatrix.com/trac/ticket/8157

**Comment by bennr01** [comment]
@adiroiban thanks, i did not see the comment on the ticket and the link.

**Comment by bennr01** [comment]
@adiroiban sorry for the delay. I made the changes.

**Comment by adiroiban** [comment]
sorry for the huge delay. Will merge. Thanks again!

Full tests are here... and will merge that branch as after all this time it required minor changes to the build system

https://github.com/twisted/twisted/pull/892

**Pull request closed without merging**